templates/gitlab/register-gitlab-component/template.yaml

```
@@ -0,0 +1,109 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
+metadata:
+  name: register-gitlab-component
+  title: Register existing component to Software Catalog from Gitlab
+  description: Registers existing component (Gitlab repository) to Software Catalog
+  tags:
+    - import
+    - catalog
+    - register
+spec:
+  owner: janus-authors
+  system: janus-idp
+  type: service
+
+  parameters:
+    - title: Provide information about the Gitlab location
+      required:
+        - gitlabHost
+        - projectOwner
+        - projectName
+      properties:
+        gitlabHost:
+          title: Gitlab hostname
```

```suggestion
          title: GitLab Hostname
```

Please add the corresponding GitHub README and template changes here.